lua/hop/dict/ascii.lua

```diff
@@ -0,0 +1,98 @@
+-- Ascii
```

Comment:
Eh, this is going to generate a regression in lots of configuration, starting with mine, where I use unicode on some of my keys (I’m using the bépo layout, so `é` is a direct key to me).

Not sure this is a good idea to do it that way.

Reply:
I think I had ignored the case for different layout. And for users who use bépo layout, the key of dict from `ascii.lua` may be unicode keys, but not only the ascii keys.
I fix the issue by rename `ascci.lua` to `zh.lua` ( and add some unicode mappings to zh.lua used in Chinese language ), meaning `zh.lua` is a basic mappings to Chinese language.